src/index.ts

```diff
@@ -1,11 +1,16 @@
 import { Union as _Union } from './union';
 import { IFS } from './fs';
+export * from './lists'
 
 export interface IUnionFs extends IFS {
-  use(fs: IFS): this;
+  use: (...args: Parameters<_Union['use']>) => this;
 }
 
 export const Union = (_Union as any) as new () => IUnionFs;
 
 export const ufs = (new _Union() as any) as IUnionFs;
 export default ufs;
+export {
```

Comment:
This export isn't needed, as far as I can tell. If it's desired for you as a consumer, probably best to do it in another PR to reduce the diff of this one.